packages/admin/admin/src/mui/menu/Item.tsx

```diff
@@ -24,26 +25,28 @@ const Item: React.FC<WithStyles<typeof styles> & MenuItemProps & MuiListItemProp
     icon,
     level = 1,
     secondaryAction,
+    isMenuOpen,
     ...otherProps
 }) => {
     const context = React.useContext(MenuContext);
```

Comment:
Why can't you use `context.open` from here?

Reply:
Because of conditional styling in the .styles file